src/core_modules/capture-core-utils/date/date.utils.js

```diff
@@ -1,17 +1,25 @@
 // @flow
-import moment from '../moment/momentResolver';
+import { moment } from '../moment';
 
 export const displayTypes = {
     SHORT: 'short',
     MEDIUM: 'medium',
     LONG: 'long',
 };
 
-export function getMonthName(monthNumber: number) {
-    const date = moment([0, monthNumber - 1]);
-    return date.format('MMMM');
+/**
+ * Some locales are using numeral glyphs other than european. This method ensures the moment format method returns a value in european glyphs
+ * @param {*} momentDate: the moment instance
+ * @param {string} format: the moment format
+ * @returns {string} A formatted string with european glyphs
+ */
+export function getFormattedStringFromMomentUsingEuropeanGlyphs(momentDate: moment$Moment, format: string) {
+    const europeanMoment = momentDate.clone();
```

Comment:
I would do this is this was possible. 
```suggestion
    const europeanMoment = momentDate.clone().locale('en');
```

Reply:
:+1 Agreed